phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexIT.java

```diff
@@ -45,19 +45,24 @@
 	private final boolean localIndex;
 	private final String tableDDLOptions;
 	
```

Comment:
Will change the check in TableRef.equals to

if ((table.getName() == null && other.table.getName() != null) 
                || (table.getName()!=null && !table.getName().getString().equals(other.table.getName().getString()))) return false;